crates/core/src/host/v8/syscall.rs

```diff
@@ -961,6 +932,17 @@ fn datastore_update_bsatn(scope: &mut PinScope<'_, '_>, args: FunctionCallbackAr
     Ok(row)
 }
 
+fn get_jwt_payload(scope: &mut PinScope<'_, '_>, args: FunctionCallbackArguments<'_>) -> SysCallResult<Vec<u8>> {
```

Comment:
Let's move this to before `fn identity` like in `wasm_instance_env`.
Also some docs conforming to the format of the other ABIs would be good.